quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/cli/PicocliTest.java

```diff
@@ -471,6 +478,9 @@ public void wildcardLevelForCategory() {
         var value = nonRunningPicocli.config.getConfigValue("quarkus.log.category.\"org.keycloak\".level");
         assertEquals("quarkus.log.category.\"org.keycloak\".level", value.getName());
         assertEquals("WARN", value.getValue());
+        value = nonRunningPicocli.config.getConfigValue("quarkus.log.category.\"org.keycloak1\".level");
+        assertEquals("quarkus.log.category.\"org.keycloak1\".level", value.getName());
+        assertNull(value.getValue());
```

Comment:
What exactly does this case check? It should pass(be true) even without these changes in this PR, right?

Reply:
It would, but some of my intermediate attempts at addressing this relied upon proving the default value when not set. We don't want to do that because that overrides any quarkus level setting. I just wanted to make sure there was a unit test covering that.